cache/sampleStoreTimeout.js

```diff
@@ -180,10 +178,12 @@ module.exports = {
 
       return redisClient.batch(sampCmds).execAsync();
     })
-    .then(() => {
-      const res = { numberEvaluated, numberTimedOut, timedOutSamples };
-      return res;
-    })
+    .then(() => Promise.all(timedOutSamples.map((s) =>
+      model.getSample({ key: { value: s.name } }))))
+    .then((samples) => Promise.all(samples.map((s) =>
+      publisher.publishSample(s, helper.associatedModels.subject,
+        sampleEvent.upd, helper.associatedModels.aspect))))
```

Comment:
Were we really not publishing on timeouts before? How does the multitable show timed-out samples? Do they get published elsewhere?

Reply:
You're right... it was too crazy to think that we weren't publishing sample timeout events... I couldn't trace the code path, but I do observe the real-time events getting sent in sandbox when a timeout happens... I'll have to revisit the code and see why they weren't getting sent in my tests.